src/structures/Tibia.ts

```diff
@@ -24,6 +24,24 @@ export class Tibia
 		return result.affected !== 0;
 	}
 
+	/* returns whether there was account for that discord id */
+	public static async chargeCoins(discordID: string, coins: number): Promise<boolean>
+	{
+		const accountRepo = getRepository(Account);
+
+		const result = await accountRepo.decrement({ discordID }, "coins", coins);
+
+		return result.affected !== 0;
+	}
+
+	public static async recordGamble(discordID: string, coins: number, won: boolean = true): Promise<boolean>
```

Comment:
I don't think that defaults make sense to be here, neither is it used anywhere.
```suggestion
	public static async recordGamble(discordID: string, coins: number, won: boolean): Promise<boolean>
```